storagenode/monitor/monitor.go

```diff
@@ -197,5 +197,11 @@ func (service *Service) AvailableBandwidth(ctx context.Context) (_ int64, err er
 		return 0, Error.Wrap(err)
 	}
 	allocatedBandwidth := service.allocatedBandwidth
-	return allocatedBandwidth - usage, nil
+	availableBandwidth := allocatedBandwidth - usage
+
+	mon.IntVal("allocated_bandwidth").Observe(allocatedBandwidth) //locked
+	mon.IntVal("usage_bandwidth").Observe(usage)                  //locked
+	mon.IntVal("available_bandwidth").Observe(availableBandwidth) //locked
```

Comment:
Is allocated_bandwidth calculated per-request, or overall? If overall, how often does this reset?
Also, perhaps naming it used_bandwidth instead of usage_bandwidth might be more clear.

Reply:
If I were to guess, this is overall bandwidth for a month. Which means it would probably reset every month. But I'm not personally familiar with this area of the codebase.

Reply:
It's overall. Based on this comment: https://github.com/storj/storj/pull/1446#discussion_r265072985

It should reset once per month.

Reply:
In that case, if we were to only report two metrics, I'd go with allocated and used.